Packer/ArkCase-EE-Core/packer.json

```diff
@@ -0,0 +1,71 @@
+{
+  "description": "Packer template to build an ArkCase FOIA AMI ready to be published on AWS Marketplace",
```

Comment:
I guess this should be "ArkCase Core", not "ArkCase FOIA"

Reply:
Yes , you're right. Thanks @fabricetriboix